src/components/steps/styles.ts

```diff
@@ -0,0 +1,269 @@
+import stitches from '../../stitches.config';
+
+const { styled } = stitches;
+
+export const StepsContainerStyle = styled('div', {
+  display: 'flex',
+});
+
+export const StepDrawStyle = styled('div', {
+  display: 'flex',
+  alignItems: 'center',
+});
+
+export const LineStyle = styled('div', {
+  width: '64.5px',
+  height: '1px',
+  backgroundColor: '#ced2db',
```

Comment:
```suggestion
  backgroundColor: '#ced2db',
```

this color not have in DS?